tensorflow/core/ops/image_ops.cc

```diff
@@ -558,6 +558,19 @@ contents: 0-D. PNG-encoded image.
 )doc");
 
 // --------------------------------------------------------------------------
+REGISTER_OP("DecodeBmp")
+    .Input("contents: string")
+    .Output("image: uint8")
```

Comment:
` Unrecognized type string 'uint32' from Attr("T: {uint8, uint16, uint32} = DT_UINT8") for Op DecodeBmp`
It seems uint32 is a valid type supported by `tensorflow/core/framework/op_def_builder.cc`. Is int32 acceptable? Or you really want to have uint32.